recipes/py-hunspell/build.sh

```diff
@@ -0,0 +1,11 @@
+
+# Fix hunalign conda package
+if [[ ! -f $PREFIX/lib/libhunspell.so ]]; then
+  ln -s $PREFIX/lib/libhunspell{-1.7,}.so
+fi
+if [[ ! -f $PREFIX/lib/libhunspell.a ]]; then
+  ln -s $PREFIX/lib/libhunspell{-1.7,}.a
+fi
```

Comment:
Hi! I totally agree. If that's possible, I'd be able to remove this code. I needed to add it since [py-hunspell](https://pypi.org/project/hunspell/) could not work without that fix for `hunspell` conda package.

Reply:
Sounds good.  If the hunspell-feedstock maintainers don't reply about this and fix, I can help you submit a PR there.  I haven't looked at your package output but I'm fairly certain that creating the symlinks here would include them in the py-hunspell package and that is going to be really confusing longer term.

Reply:
This does look like something that should be fixed in hunspell itself. Is there a reason we need the non-versioned names? Also does this need to be done on Mac too?

Reply:
Well, this is something I don't know how it works exactly, but when you install hunspell with apt, `/usr/lib/x86_64-linux-gnu/libhunspell-1.7.so.0.0.1`, at least, is created. I don't know why when the manual compilation is done (like I have checked out that is done in https://github.com/conda-forge/hunspell-feedstock/blob/main/recipe/build.sh), you don't get the versioned libraries and get the non-versioned instead.

When I run `sudo find / -type f | grep libhunspell 2> /dev/null` (in case it is useful somehow):

```
/usr/share/doc/libhunspell-1.7-0/changelog.Debian.gz
/usr/share/doc/libhunspell-1.7-0/copyright
/usr/lib/x86_64-linux-gnu/libhunspell-1.7.so.0.0.1
/var/lib/dpkg/info/libhunspell-1.7-0:amd64.list
/var/lib/dpkg/info/libhunspell-1.7-0:amd64.triggers
/var/lib/dpkg/info/libhunspell-1.7-0:amd64.md5sums
/var/lib/dpkg/info/libhunspell-1.7-0:amd64.shlibs
```

Reply:
Might be the problem related to the fact that in https://github.com/conda-forge/hunspell-feedstock/blob/main/recipe/build.sh is not being invoked `ldconfig` as is indicated in https://github.com/hunspell/hunspell. If I'm not wrong, `ldconfig` should add the libraries which we are talking about, but I don't know how `ldconfig` should be invoked since the scope should be the conda environment instead of the system.

I have checked some issues and there are examples where these libraries are created indeed (e.g. https://github.com/hunspell/hunspell/issues/701), so it seems that it is not a problem of hunspell itself, but the recipe (I might be wrong, I'm just guessing).

Reply:
I would have thought the `compiler` bit in the build dependencies would install the right tools to make ldconfig do the right thing.

Reply:
Is it being considered to take any action about this in https://github.com/conda-forge/hunspell-feedstock?

Reply:
@timsnyder what should be the next step? I'm not sure if this problem is being/going to be addressed in hunspell feedstock